package NYT.NChaosServer.NProto;

import "yt/server/master/chaos_server/proto/alien_cell.proto";

////////////////////////////////////////////////////////////////////////////////

message TReqUpdateAlienCellPeers
{
    repeated TAlienCellConstellation constellations = 1;
    optional bool full_sync = 2 [default = false];
}

////////////////////////////////////////////////////////////////////////////////

